seq2science/rules/qc.smk

```diff
@@ -878,15 +878,16 @@ def get_peak_calling_qc(sample):
 
     # deeptools profile
     assembly = treps.loc[sample, "assembly"]
+    narrowpeak_used = get_peak_ftype(list(config["peak_caller"].keys())[0]) == "narrowPeak"
```

Comment:
`"narrowPeak" in [(get_peak_ftype(pc) for pc in list(config["peak_caller"].keys())]` maybe?

Reply:
:+1: